requirements/extras/test_requirements.txt

```diff
@@ -39,3 +39,5 @@ tritonclient[http]<2.37.0
 onnx==1.14.1
 # tf2onnx==1.15.1
 nbformat>=5.9,<6
+accelerate
+huggingface_hub
```

Comment:
lets pin all these dependencies to avoid surprises due to auto consumption of these.

Reply:
Not sure what pin means, asked on pySDK channel

Reply:
Thanks addressed